### PR TITLE
Add Malayalam language.

### PR DESCRIPTION
**Pull request by arunshekher**
Update translationMetadata.json to add the language Malayalam.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @arunshekher

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant/frontend#13854).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by bramkragten** [comment]
Hi! Can you send me your username in Localize so I can add you as editor for the Malayalam language?

**Comment by arunshekher** [comment]
> Hi! Can you send me your username in Localize so I can add you as editor for the Malayalam language?

Hi Bram, I've just sent you an email with my Lokalize account email id to your email in your Github profile.

**Comment by arunshekher** [comment]
Hi Bram, Hope you got my email with my Lokalise account details. Is there anything else I should be doing? Thanks

**Review by bramkragten — DISMISSED** [review]

**Comment by bramkragten** [comment]
> Hi Bram, Hope you got my email with my Lokalise account details. Is there anything else I should be doing? Thanks

Hi, please fix the linting issue (run prettier, or remove the two extra spaces on line 121)

File: src/translations/translationMetadata.json [hunk 1]
```diff
@@ -116,6 +116,9 @@
   "lv": {
     "nativeName": "Latviešu"
   },
+  "ml": {
+    "nativeName": "മലയാളം"
+  },
```
Comment (bramkragten) [inline src/translations/translationMetadata.json:121]:
```suggestion
  },
```

**Review by arunshekher — COMMENTED** [review]
Committed the suggestions and resubmitting.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**